### PR TITLE
[v6r19] New logging backend: MessageQueueBackend

### PR DESCRIPTION
**Pull request by aldbr**
The pull request contains a new Backend and its documentation, used to send logs to a MessageQueue.

### CODE REVIEW

File: FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  """
+
+  def __init__(self, queue):
+    """
+    Initialization of the MessageQueueHandler.
+
+    :params queue: string representing the queue identifier in the configuration.
+                   example: "mardirac3.in2p3.fr::Queue::TestQueue"
+    """
+    logging.Handler.__init__(self)
```
Comment (fstagni) [inline FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py]: can't you use `super` here?

Comment (aldbr) [inline FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py]: Honestly, I don't know why it was not the case before. 
Thank you

**Comment by atsareg** [comment]
Do you have a receiver for these logging messages ? This is a potentially interesting backend but what is a presumed use case ? What happens if we enable this backend by default - the MQ will start to receive zillions of messages and can suffer overflow of internal buffers because no consumer is poping these messages from the queue.

**Comment by chaen** [comment]
The receiver can be anything, not even DIRAC related. If some one enables that and do not have a consumer, too bad :-)

**Pull request merged**